src/worker/link-storage-to-kmp.ts

```diff
@@ -0,0 +1,19 @@
+import { WordListFromArray } from "@predictive-text-studio/lexical-model-compiler";
+import { generateKmp } from "./generate-kmp";
+import Storage from "./storage";
+
+export async function linkStorageToKmp(storage: Storage): Promise<ArrayBuffer> {
+  const sourcesFromDB = await storage.fetchAllFiles();
+  const sources: WordListFromArray[] = [];
+  for (const { name, wordlist } of sourcesFromDB) {
+    sources.push(new WordListFromArray(name, wordlist));
+  }
```

Comment:
If I understand this correctly, these lines of code take each `StoredWordList` from the DB and convert it to a `WordListFromArray` to append to sources, which is later used to generate the kmp.

If so, I would probably move this to another helper function like:
```
let sources: WordListFromArray[] = generateSourcesFromStorage(storage);
``` 

to make `linkStorageToKmp` more readable/decouple functionality.

Reply:
@Xuechunqiu could you please factor out `generateSourcesFromStorage()` as Pranav suggested?